.github/workflows/hackage.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   check-and-upload-tarballs:
     runs-on: ubuntu-latest
     strategy:
-      fail-fast: ${{ !contains(github.ref_name, 'check') && github.event.inputs.upload-candidates != 'true' }}
+      fail-fast: ${{ github.event.inputs.upload-candidates == 'true' }}
```

Comment:
Not sure what any of this does but just making sure `!=` is supposed to change to `==`?

Reply:
Thanks for noting it. Yeah it was wrong and this fixes it.